cmd/acra_configui/acra_configui.go

```diff
@@ -172,12 +210,13 @@ func main() {
 	} else {
 		logging.SetLogLevel(logging.LOG_VERBOSE)
 	}
-
+	parseTemplate()
+	configParamsBytes = parseConfig()
 	http.HandleFunc("/index.html", index)
 	http.HandleFunc("/", index)
-	http.Handle("/static/", http.StripPrefix("/static/", http.FileServer(http.Dir("./static"))))
+	http.Handle("/static/", http.StripPrefix("/static/", http.FileServer(http.Dir(*staticPath))))
 	http.HandleFunc("/acraserver/submit_setting", SubmitSettings)
-	log.Info(fmt.Sprintf("AcraConfigUI is listening @ :%d with PID %d", *port, os.Getpid()))
-	err := http.ListenAndServe(fmt.Sprintf(":%d", *port), nil)
+	log.Info(fmt.Sprintf("AcraConfigUI is listening @ %s:%d with PID %d", *host, *port, os.Getpid()))
```

Comment:
you can use just `log.Infof` with placeholders without `fmt.Sprintf`

Reply:
done